packages/app-utils/src/components/ConnectionBootstrap.tsx

```diff
@@ -83,7 +160,34 @@ export function ConnectionBootstrap({
     [connection]
   );
 
-  if (connection == null || error != null) {
+  if (isReconnecting && !isShutdown) {
+    return (
+      <DebouncedModal isOpen={isReconnecting} debounceMs={1000}>
+        <InfoModal
+          icon={vsDebugDisconnect}
+          title={
+            <>
+              <LoadingSpinner /> Attempting to reconnect...
+            </>
+          }
+          subtitle="Please check your network connection."
+        />
+      </DebouncedModal>
+    );
+  }
+
+  if (isAuthFailed) {
+    return (
+      <BasicModal
+        confirmButtonText="Refresh"
+        onConfirm={handleRefresh}
+        isOpen={isAuthFailed}
+        headerText="Authentication failed"
+        bodyText="Credentials are invalid. Please refresh your browser to try and reconnect."
+      />
+    );
+  }
```

Comment:
I believe both of these cases should actually render the children still.

@mofojed we don't want the children completely gone when trying to reconnect, right?

This PR
![image](https://github.com/deephaven/web-client-ui/assets/8619789/b36d1f91-0479-4099-9b0c-25a900165f38)

Prior to this PR
![image](https://github.com/deephaven/web-client-ui/assets/8619789/95b49d9f-985d-4cb5-bd98-c8de04b0e3a6)

Reply:
Yes children should still render